openhcl/underhill_attestation/src/lib.rs

```diff
@@ -295,9 +294,12 @@ pub async fn initialize_platform_security(
             driver,
         )
         .await
-        .map_err(ErrorInner::RequestVmgsEncryptionKeys)?
+        .map_err(|e| {
```

Comment:
tracing from within a map_err call is almost never what you want. the error variant itself should have all the necessary context, rather than having disjoint context between a tracing statement + the downstream error-log statement.